src/Microsoft.Owin.Security.ActiveDirectory/IssuerSigningKeys.cs

```diff
@@ -19,6 +19,6 @@ internal class IssuerSigningKeys
         /// <summary>
         /// Signing tokens.
         /// </summary>
-        public IEnumerable<X509SecurityToken> Tokens { get; set; }
+        public IEnumerable<SecurityKey> Keys { get; set; }
```

Comment:
@Tratcher my apologizes. This is one of the best commits I've ever read. But I can't find any reference/article for migration from System.IdentityModel to Microsoft.IdentityModel Could you please provide us more information about this? We've exciting code that needs to migrated